__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/development-data/index')
+
+
+beforeEach(()=> seed(testData))
+afterAll(()=>db.end())
+
+describe('GET /api/topics', () => {
+  test('responds with an array of topic objects', () => {
+      return request(app)
+          .get('/api/topics')
+          .expect(200)
+          .then(({ body }) => {
+              expect(body.topics).toBeInstanceOf(Array);
```

Comment:
Great test. As it's gonna reseed before each test, so the array won't change, thus, it would be better to check the length of the array as well